servers/rendering/renderer_rd/renderer_compositor_rd.cpp

```diff
@@ -175,5 +175,20 @@ RendererCompositorRD::RendererCompositorRD() {
 
 	storage = memnew(RendererStorageRD);
 	canvas = memnew(RendererCanvasRenderRD(storage));
-	scene = memnew(RendererSceneRenderImplementation::RenderForwardClustered(storage));
+
+	// !BAS! should we move the definition somewhere else?
+	String mode = GLOBAL_DEF("rendering/vulkan/rendering/mode", "Forward clustered");
```

Comment:
This should be moved to rendering_server.cpp here:
https://github.com/godotengine/godot/blob/0ee744ff5f3cf045a2699aa4111de3200b9d70be/servers/rendering_server.cpp#L2285

And it should replace ``use_low_end_renderer``

Also it should use ``GLOBAL_DEF_RST`` so that it suggests restarting the editor for the user